BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/location/provider/LocationPredictorTest.java

```diff
@@ -19,45 +19,80 @@
 
     public static final double HUMAN_WALKING_SPEED = 1.388889; // meters per second
 
-    public static final Location SQUARE_ONE_GENDARMENMARKT = new Location(52.513588, 13.392995);
-    public static final Location SQUARE_TWO_GENDARMENMARKT = new Location(52.513632, 13.392986);
-    public static final Location SQUARE_THREE_GENDARMENMARKT = new Location(52.513675, 13.392979);
-    public static final Location SQUARE_FOUR_GENDARMENMARKT = new Location(52.513720, 13.392971);
-    public static final Location SQUARE_EXPECTED_GENDARMENMARKT = new Location(52.513764, 13.392962);
+    public static final Location LINE_ONE_GENDARMENMARKT = new Location(52.513588, 13.392995);
```

Comment:
A line associated with a single coordinate doesn't seem right. Rename them to make them understandable.